src/iodide-api/flow.js

```diff
@@ -0,0 +1,35 @@
+
+let explicitResolutionStatusFlag
```

Comment:
Ah sorry, must have been a miscommunication -- i think we're better off using the redux store for global app state pretty much whenever we can. There will of course be a handful of situations when that's not possible, but I think in general we should try to stick to that discipline unless there is a compelling/unavoidable reason not to.

Reply:
yeah, tbh the more I think about it, the better off we'll be by not managing state privately, even if it's done cleanly / concisely. Otherwise we're forced to maintain additional mental models, which doesn't seem worth it. I'll make the changes today.

I'm now wondering if the global evaluation queue should also be stored in Redux vs. privately managed in `actions.js` for the same reason.

Reply:
Yes, it's a fair question. I think the difference here is that:
 - `explicitResolutionStatusFlag` is a flag (really an enum IIRC), which is simply serializable (type->string) and which will benefit from the schema validation and centralization that comes from placing it in the redux store. Also, it's something that we mutate and peek at, and a bit of state that other app behavior changes based on.
- the globalEvalQueue, on the other hand (if i have a correct understanding of how it's implemented), is more of like a _singleton object_ than an chunk of _app state_ that other app logic depends on.

does this distinction seem sane? there will be grey areas sometimes, but this is more or less how i've been approaching trying to think about where stuff goes.

Reply:
Yep, that seems right. `explicitResolutionStatusFlag` actually functions / is tested as an enum (functionally), so moving that over will be pretty easy. The question for me is, should this simply be a flag that exists on the top level of the store, or in the cell itself. The reason why I didn't have it in the store originally was because it seemed like the easiest way of detecting if a user called for explicit continuation within a code block, and wasn't really part of the store. So as a temporary flag useful only for checking after `eval` it didn't seem worthwhile to maintain that in the store.